cves/kernel/CVE-2017-1000252.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: An vulnerability was found within the linux kernal which allowed guest users , thus, causing a possible crash or denial of service.
```

Comment:
All of the CVE's were found in the linux kernel*. This needs slightly more detail. A nonSE wouldn't know what you're talking about.

Reply:
It would be better, if more detail was provided about how guest users being allowed resulted in DoS.

Reply:
Added more information thanks for checking